src/lib/reasoners/satml_frontend.ml

```diff
@@ -92,12 +92,28 @@ module Make (Th : Theory.S) : Sat_solver_sig.S = struct
   let empty_with_inst add_inst =
     { (empty ()) with add_inst = add_inst }
 
-  exception Sat of t
+  (* Leaving this exception in case we actually raise Sat one day.
+     Remember to update `safe_call` by catching this exception if you do. *)
```

Comment:
`safe_call` already catches this exception?

Reply:
Oops, I removed the Sat exception, wrote this comment, put it back and forgot about it.